tests/integ/test_pandas_to_df.py

```diff
@@ -160,7 +162,12 @@ def test_write_pandas(session, tmp_table_basic):
 
     df = session.write_pandas(pd, tmp_table_basic, overwrite=True)
     results = df.to_pandas()
-    assert_frame_equal(results, pd, check_dtype=False)
+    if local_testing_mode:
+        assert_frame_equal(results, pd, check_dtype=False)
+        # we stop the test here for local testing as the following is testing temp table behavior
+        return
```

Comment:
Instead of returning, let's put the rest of the test code in a if clause.

Reply:
updated